### PR TITLE
Feat content fade

### PR DESCRIPTION
**Pull request by mikeyrayvon**
- Fade text while panning
- Fade map content while hovering text
- Panning takes priority over hovering text
  - i.e While panning AND hovering, text content fades, map content doesn't

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -513,6 +514,68 @@ Site.Map = {
...
+    var _this = this;
+
+    _this.handleMapPanning();
+    _this.handleTextHover();
+  },
+
+  handleMapPanning: function() {
+    var _this = this;
+
+    map.addEventListener('startpan', function() {
```
Comment (mixcas) [inline src/js/main.js]: This works, but how? Wheres `map` coming from? D;

Comment (mixcas) [reply src/js/main.js]: @mikeyrayvon ?

Comment (mikeyrayvon) [reply src/js/main.js]: idk?

Comment (mikeyrayvon) [reply src/js/main.js]: spooky! 👻

**Comment by mikeyrayvon** [comment]
fuck i did that that in the wrong branch but this closes #16

**Comment by mikeyrayvon** [comment]
and that commit message is wrong fuck. should be `get map elem for fade content map events`

**Comment by mikeyrayvon** [comment]
forgive meee

File: src/js/main.js [hunk 2]
```diff
@@ -539,10 +604,9 @@ Site.Minimap = {
   handleClick: function(elem) {
     var _this = this;
 
-    // Parse targets position JSON from data attr
-    var pos = JSON.parse($(elem).attr('data-pos'));
-    var col = pos.col;
-    var row = pos.row;
+    // Get grid position from target elem data attrs
+    var col = $(elem).attr('data-col');
```
Comment (varffs) [inline src/js/main.js:597]: v bad behaviour lol

**Review by varffs — COMMENTED** [review]
This works well as code, but I think is a bad idea as a feature. So I'll merge and then the client can see. But I think we should also lobby to remove this feature.

**Pull request merged**